docs/content/latest/api/ysql/exprs/window_functions/function-syntax-semantics/first-value-nth-value-last-value.md

```diff
@@ -2,7 +2,7 @@
 title: first_value(), nth_value(), last_value()
 linkTitle: first_value(), nth_value(), last_value()
 headerTitle: first_value(), nth_value(), last_value()
-description: The SQL window functions first_value(), nth_value(), last_value()
+description: Describes the functionlity of the YSQL window functions first_value(), nth_value(), last_value().
```

Comment:
```suggestion
description: Describes the functionlity of the YSQL window functions first_value(), nth_value(), and last_value().
```

Reply:
Same typo in others below.

Reply:
Typo fixed globally. Omission of "and" fixed.